linux/keyman-config/keyman_config/ibus_util.py

```diff
@@ -130,6 +132,7 @@ def restart_ibus(bus=None):
         except Exception as e:
             logging.warning("Failed to restart IBus")
             logging.warning(e)
+    time.sleep(1)  # 1s
```

Comment:
Can you add a comment explaining this delay?

Reply:
done